index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Merriweather:900|Roboto">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <section>
+        <article id="article1">
+            
+            <h1>Život u Splitu</h1>
+            <br>
+            <p>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eusmod tempor
+                incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud
+                exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Aed do
+            </p>
+            <div class = "div1">
+                <button>Građani</button>
+                <button>Edukacija i odgoj</button>
+                <button>Stanovanje i gradnja</button>
+                <button>Kultura</button>
+            </div>
+            <div class = "div2">
+                <button>Sport</button>
+                <button>Promet</button>
+                <button>Naknade i porezi</button>
+                <button>Gradski kotari i mjesni odbori</button>
+            </div>
+        </article>
+        <article id="article2">
```

Comment:
You really want to avoid using ids for styling purposes, the use case is very rare. This should be targeted with classes :)